src/Features/Core/Portable/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -57,11 +57,19 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
 
             var installerService = GetPackageInstallerService(document);
             var packageSources = searchNuGetPackages && symbolSearchService != null && installerService?.IsEnabled(document.Project.Id) == true
-                ? installerService.GetPackageSources()
+                ? await installerService.TryGetPackageSourcesAsync(allowSwitchToMainThread: false, context.CancellationToken).ConfigureAwait(false)
                 : ImmutableArray<PackageSource>.Empty;
 
+            if (packageSources is null)
```

Comment:
#nullable enable the file so we have the tracking for nullable value types too?

Reply:
I'm leaving this one for a future change